src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -46,6 +46,10 @@ public void addSection(Section section) {
         this.sections.addSection(section);
     }
 
+    public void deleteSection(Station deletedStation) {
```

Comment:
삭제된 station이 아니라 삭제할 station이고 메서드명에서 충분히 삭제 의미를 전달하고 있으니 변수명에 중복으로 표시할 필요는 없겠네요.

```suggestion
    public void deleteSection(Station station) {
```